src/authentication/src/Revoked.jsx

```diff
@@ -30,7 +30,8 @@ class Revoked extends Component {
     }
   }
 
-  render({ t }) {
+  render() {
+    const { t } = this.props
     return (
       <Modal
         title={t('mobile.revoked.title')}
```

Comment:
I think you should add contextType here ?

Reply:
I think you are talking of the contextType to have the `t` ? These are automatically added by I18n.

Reply:
Nop sorry. We receive also the client from the context `  const cozyClient = this.context.client `

So we have to add client to contextType (as you have done https://github.com/cozy/cozy-drive/pull/1287/files#diff-503680972da6775577d2afa740206f70R106)

Reply:
Done, thx. https://github.com/cozy/cozy-drive/pull/1287/commits/b6e5db220f643a70acc002ed778aee7c2541db23